src/runtime/ProxyObject.h

```diff
@@ -78,7 +78,10 @@ class ProxyObject : public Object {
     // http://www.ecma-international.org/ecma-262/5.1/#sec-8.6.2
     virtual const char* internalClassProperty() override
     {
-        return "Proxy";
+        if (!m_target) {
+            return "Null";
```

Comment:
What is the expected behavior when m_target is nullptr? Shouldn't it be an exception?

Reply:
Proxy [revoke](https://www.ecma-international.org/ecma-262/6.0/#sec-proxy-revocation-functions) method invalidates Proxy object and its `target` value is set to null. As far as I know, no other way except `revoke` can make the `target` value as null. IMO when `target` is null, we should throw an exception because every access to the invalidated Proxy object triggers an exception each.